lib/verifier.rb

```diff
@@ -120,4 +120,54 @@ def verify_file filename
     verify manifest
   end
 
+  def check_links filename
+    puts 'Verify links...'
+    manifest = Manifest.parse_file filename
+
+    if manifest.urls.homepage
+      verify_link('homepage',manifest.urls.homepage)
+    end
+    if manifest.urls.api_docs
+      verify_link('api_docs',manifest.urls.api_docs)
+    end
+    if manifest.urls.download
+      verify_link('download',manifest.urls.download)
+    end
+    if manifest.urls.tutorial
+      verify_link('tutorial',manifest.urls.tutorial)
+    end
+    if manifest.urls.vcs
+      verify_link('vcs',manifest.urls.vcs)
+    end
+    if manifest.urls.description_source
+      verify_link('description_source',manifest.urls.description_source)
+    end
+    if manifest.urls.announcement
+      verify_link('announcement',manifest.urls.announcement)
+    end
+    if manifest.urls.mailing_list
+      verify_link('mailing_list',manifest.urls.mailing_list)
+    end
+    if manifest.urls.contact
+      verify_link('contact',manifest.urls.contact)
+    end
```

Comment:
I am not sure if you are meaning removing this type of a duplication.
url = manifest.urls.contact
verify_link('contact',url) if !url.nil?